VTEX - Ad Network API.json

```diff
@@ -608,6 +617,399 @@
+                                        "description": "Merchant's account name. Should be the same as the account name registered at VTEX.",
+                                        "example": "apiexamples"
+                                    },
+                                    "MacId": {
+                                        "type": "string",
+                                        "description": "A long-life identifier stored in the browser cookies or device of each shopper. It is valid for one year and is renewed as new events are sent. It expires if the user clears their cookies or deletes the app. Use UUID Version 4.",
+                                        "example": "f805a89b-ef06-4d6a-ae9d-4dd5699ccd6f"
+                                    },
+                                    "Url": {
+                                        "type": "string",
+                                        "description": "Current page URL or the current mobile view.",
+                                        "example": "https://www.apiexamples.com.br/busca?tshirt"
+                                    },
+                                    "Ref": {
+                                        "type": "string",
+                                        "description": "Previous page URL (accessed before the current page URL) or mobile equivalent.",
+                                        "example": "https://www.apiexamples.com.br"
+                                    },
+                                    "SessionId": {
+                                        "type": "string",
+                                        "description": "Session identifier, which expires after 30 minutes. Use UUID Version 4.",
+                                        "example": "fb252d90-7494-4097-90fd-cc73eeaa575c"
+                                    },
+                                    "placement": {
+                                        "type": "string",
+                                        "description": "Refers to where the ad is shown on the site.",
+                                        "enum": [
+                                            "top_search",
+                                            "middle_search",
+                                            "home_shelf",
+                                            "pdp_shelf",
+                                            "search_shelf",
+                                            "cart_shelf",
+                                            "plp_shelf"
+                                        ],
+                                        "example": "top_search"
+                                    },
+                                    "position": {
+                                        "type": "number",
+                                        "description": "Position of the ads in the list.",
+                                        "example": 2
```

Comment:
Maybe it would be worth using an example that didn't seem like an integer just to make it more clear.